File: intro-dev-tsjs-2024-03-28/6.ts
import ollama from 'ollama';

const msgs = [
  {"role": "system", content: "The user will give you a concept. Explain it to a 5 year old, using descriptive imagery and interesting and fun stories."},
  { "role": "user", content: "Quantum Physics" }
]
const output = await ollama.chat({ model: "llama2", messages: msgs })

console.log(output.message.content)

File: 2024-04-01-intro-dev-python/6.py
import ollama 

msgs = [
  {"role": "system", "content": "The user will give you a concept. Explain it to a 5 year old, using descriptive imagery and interesting and fun stories."},
  { "role": "user", "content": "Quantum Physics" }
]
output = ollama.chat(model="llama2", messages=msgs )

print(output['message']['content'])